### PR TITLE
Afternoon planning

### PR DESCRIPTION
**Pull request by dcwalk**
Hey @garrying here is my preliminary planning for both afternoon sessions. I've roughed you in as moderator 😄 📶

### CODE REVIEW

File: sessions/session-e.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Session E: A History of Toronto's Networks
```
Comment (garrying) [inline sessions/session-e.md:1]: Are we following a style convention with the way these sessions are titled? A good title will be helpful frame the discussion, or segue into it.

File: sessions/session-e.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- Front panel with chairs
+- Semi-circular seating with centre ring for [fishbowl](https://www.facinghistory.org/resource-library/teaching-strategies/fishbowl)
+
+## Registration
+
+40
+
+## Presenter(s) and Bio
+
+[To be filled in]
```
Comment (garrying) [inline sessions/session-e.md:41]: Is a bio needed from me?

File: sessions/session-e.md (hunk 1 above)
Comment (dcwalk) [reply sessions/session-e.md:1]: The morning sessions have a short title ":" then longer.
These two afternoons don't right now.

I've been working with the ambiguity of "network" throughout the event, and I think the perspectives in the panel could open that up. If you want to adjust the title please throw suggestions up!

File: sessions/session-e.md (hunk 2 above)
Comment (dcwalk) [reply sessions/session-e.md:41]: Yeah as moderator that'd be great!

**Comment by dcwalk** [comment]
I'm going to merge this for now, but @garrying please make adjustments to your session (esp. name?) and either open a PR or ping me to discuss?

File: sessions/session-e.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+35 mins moderated fishbowl conversation
+
+## Materials
+
+- Presentation Laptop and projector adaptor
+
+## Format
+
+- Front panel with chairs
+- Semi-circular seating with centre ring for [fishbowl](https://www.facinghistory.org/resource-library/teaching-strategies/fishbowl)
```
Comment (benhylau) [inline sessions/session-e.md:33]: Cool :)

**Pull request merged**